bindings/python/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ project(bindings)
 find_package(PCL COMPONENTS common REQUIRED)
 find_package(pybind11)
 
-pybind11_add_module(pcl common/py_point_types.cpp)
+pybind11_add_module(pcl ${CMAKE_CURRENT_SOURCE_DIR}/pybind11-gen/common/include/pcl/impl/point_types.cpp)
```

Comment:
This file needs to be generated by another tool, right?

Reply:
`generate.py` will generate this file.
This CMakeLists is used for testing if the source compiles, no pybind11 related errors.

A more general version in #25:
https://github.com/divmadan/pcl/blob/86904e45a14611c296da483b5ce532a2e4505767/python/package/CMakeLists.txt#L7-L21